src/game/common/rts/playerlist.h

```diff
@@ -44,10 +44,19 @@ class PlayerList : public SubsystemInterface, public SnapShot
         return m_local;
     }
 
+    Player *Get_Nth_Player(int i)
+    {
+        if (i < 0 || i >= 16) {
```

Comment:
`i >= ARRAY_SIZE(m_players)`

Reply:
It will actually be a enum once we add it, not yet sure what to call it, will add a todo noting that tho